c2corg_ui/templates/helpers/view.html

```diff
@@ -85,9 +85,10 @@ <h4 translate>No maps associated</h4>
 
 <%def name="show_areas(document)">\
   % if document.get('areas'):
+  <% areas = document.get('areas') %>\
```

Comment:
It is clearer to indent all the content of the ``ìf`` block, for reading reasons.